vmdb/product/toolbars/conditions_center_tb.yaml

```diff
@@ -13,5 +13,5 @@
     :items:
     - :button: condition_new
       :image: new
-      :text: 'Add a New #{ui_lookup(:model=>@sb[:folder])} Condition'
-      :title: 'Add a New #{ui_lookup(:model=>@sb[:folder])} Condition'
\ No newline at end of file
+      :text: 'Add a New #{@sb[:folder].upcase == "VM" ? ui_lookup(:model=>@sb[:folder]).upcase : ui_lookup(:model=>@sb[:folder])} Condition'
```

Comment:
@GregP Since UI lookup has vm/template split issues with this string, we can keep the logic here, but perhaps it would be better to save a method call and use this on both lines changed:
```
@sb[:folder].upcase == 'VM' ? 'VM' : ui_lookup(:model=>@sb[:folder])
```